test.sh.in

```diff
@@ -266,34 +269,105 @@ waitForAPI() {
   fi
 }
 
+######################################
+#    OVA POST SMOKE TEST RELATED     #
+######################################
+portForwarding(){
```

Comment:
portForwarding will still be needed even if OVA has public IP.

Unless we run tests in OVA vm but now current vmslaves.
But then the test env setting will be complicated.